cypress/fixtures/documents.ts

```diff
@@ -17,22 +22,23 @@ export const GRAASP_DOCUMENT_ITEM: DocumentItemType = {
   extra: buildDocumentExtra({
     content: '<h1>Some Title</h1>',
   }),
-};
+});
 
-export const GRAASP_DOCUMENT_BLANK_NAME_ITEM: DocumentItemType = {
-  id: 'ecafbd2a-5688-12eb-ae91-0242ac130004',
-  type: ItemType.DOCUMENT,
-  name: '  ',
-  description: 'a description for graasp text',
-  path: 'ecafbd2a_5688_12eb_ae93_0242ac130002',
-  settings: {},
-  createdAt: '2021-08-11T12:56:36.834Z',
-  updatedAt: '2021-08-11T12:56:36.834Z',
-  creator: CURRENT_USER,
-  extra: buildDocumentExtra({
-    content: '<h1>Some Title</h1>',
-  }),
-};
+export const GRAASP_DOCUMENT_BLANK_NAME_ITEM: DocumentItemType =
+  DocumentItemFactory({
```

Comment:
The advantage is that if one day the structure has to change, it is done from sdk so it will propagate properly in all factories. Here the `lang` property was missing. 
Also, you can omit unnecessary data (I didn't do it here) like created at, updated at, that will be automatically filled for you.